gradio/blocks.py

```diff
@@ -158,15 +158,23 @@ def get_template_context(self):
 
 
 class Column(BlockContext):
-    def __init__(self, visible: bool = True, css: Optional[Dict[str, str]] = None):
+    def __init__(
+        self,
+        visible: bool = True,
+        css: Optional[Dict[str, str]] = None,
+        variant: str = "default",
```

Comment:
We've been using the parameter "type" for most of our components, so I suggest doing that here, at least in the user facing API

```suggestion
        variant: type = "default",
```